relay-config/src/config.rs

```diff
@@ -497,6 +497,10 @@ struct Metrics {
     ///
     /// Defaults to `true`.
     buffering: bool,
+    /// Emitted metrics will be aggregated to optimize bandwidth.
+    ///
+    /// Defaults to `false`.
```

Comment:
Why do we not enable it by default?

Reply:
Also should probably document that it takes precedence over `buffering`.

Reply:
I think we can. Originally this PR was set out to be a bit more conservative in how this would be rolled out, but now that it's live in Rust consumers we can also just enable it unconditionally. In that case, would we want to keep the other flags (`buffering` etc) too, or just remove all of those different configs?

Reply:
yeah sounds good, for now i made the minimal change and just flipped the default. if all goes well i think both options (aggregation _and_ buffering) should go away, to be honest. i don't see the value in having that be configurable